terraform/autoscaling/modules/test/autoscaling.tf

```diff
@@ -0,0 +1,44 @@
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-trusty-14.04-amd64-server-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
+
+resource "aws_launch_configuration" "launch_configuration" {
+  name          = "${var.prefix}-launch-configuration"
+  image_id      = data.aws_ami.ubuntu.id
+  instance_type = "t2.micro"
```

Comment:
Can we use something newer like t3 or t4?

Reply:
Sure, but since the desired count is `0` is shouldn't matter at all.

Reply:
Updated nevertheless, should familiarize me anyhow with the new types out in the world :)